browse/templates/base.html

```diff
@@ -37,7 +57,12 @@
               },
 
               account: {
-                  id:           'ACCOUNT-UNIQUE-ID' // Highly recommended
+                  {% if session.get("institution") %}
+                  name:            '{{ session.get("institution") }}',
+                  {% endif %}
+
+                  id:              '{{ session.get("institution_id") if session.get("institution_id") else "ACCOUNT-UNIQUE-ID" }}'
```

Comment:
Do we really need to put ACCOUNT-UNIQUE-ID here or is that just documentation from Pendo to indicate the kind of thing to put as the value here?

Reply:
It has been deployed that way in other places, and shows up in pendo's ui. Seems fine to remove or keep.

Reply:
It looks like a value for visitor_id is required, but account_id is not. I think there's value in having Pendo account pages show how many visitors are not associated with an account/institution, so I think we should keep some constant here. We could add another localhost type of value to the subscription db tables or set it as -1. That seems cosmetic, so I think we leave it with their default.